diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter3/LineLengthWarnTest.kt

```diff
@@ -0,0 +1,214 @@
+package org.cqfn.diktat.ruleset.chapter3
+
+import com.pinterest.ktlint.core.LintError
+import org.cqfn.diktat.common.config.rules.RulesConfig
+import org.cqfn.diktat.ruleset.constants.Warnings.LONG_LINE
+import org.cqfn.diktat.ruleset.rules.DIKTAT_RULE_SET_ID
+import org.cqfn.diktat.ruleset.rules.LineLength
+import org.cqfn.diktat.util.lintMethod
+import org.junit.Test
+
+class LineLengthWarnTest {
```

Comment:
Please add test case with URL in markdown, like 'check this \[link\]\(http://url...)'.